analytics_data_api/v0/views/courses.py

```diff
@@ -665,7 +665,11 @@ def get_queryset(self):
 GROUP BY module_id;
         """
 
-        connection = connections[settings.ANALYTICS_DATABASE]
+        # If possible, get database from router
+        router_db = router.db_for_read(ProblemResponseAnswerDistribution)
+        database = router_db if router_db else settings.ANALYTICS_DATABASE
```

Comment:
I think the router has to return something in django or you keep going to deeper routers until you get the default so this may not be needed

Reply:
Oh yep, I think you're right. I will remove that piece.